<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signal_Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="gridFrame">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Filter Parameters</string>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>140</y>
           <width>451</width>
           <height>261</height>
          </rect>
         </property>
         <property name="title">
          <string>Sound Generation</string>
         </property>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>199</y>
            <width>318</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>411</width>
            <height>164</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Frequency 1</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>frequency 2</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>time</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>exponential</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="text">
                  <string>200</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_5">
                 <property name="text">
                  <string>200</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_6">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_7">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>generate 2 sinsounidal</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>generate 2 sinsounidal and expo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>190</y>
            <width>411</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title">
           <string>Sum Of an array Using C</string>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>411</width>
             <height>25</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Array I/P</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_8"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>Sum Up</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>411</width>
           <height>107</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>1st </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>2nd</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>3rd</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>420</y>
           <width>341</width>
           <height>101</height>
          </rect>
         </property>
         <property name="title">
          <string>Sampling signals</string>
         </property>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>sine</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>cosine</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_7">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>expo</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>470</x>
           <y>20</y>
           <width>341</width>
           <height>491</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Un-Filtered Signal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlotWidget" name="graphicsView"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Filtered Signal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlotWidget" name="graphicsView_2"/>
          </item>
         </layout>
        </widget>
        <zorder>layoutWidget</zorder>
        <zorder>layoutWidget</zorder>
        <zorder>groupBox_2</zorder>
        <zorder>groupBox_3</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
